scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -987,6 +987,61 @@ def publish_visibility(self) -> PublishVisibility:
         return PublishVisibility(visibility) if visibility else None
 
 
+class GenericAssetConfig(Config):
+    """Generic Asset Config class."""
+
+    """
+    Example:
+
+    # Remote storage path for asset data
+    remote_path:
+        type: azureblob
+        storage_name: my_storage
+        container_name: my_container
+        container_path: foo/bar
+    # Local folder path to copy remote files into
+    local_path: ./prompt
+
+    """
+
+    def __init__(self, file_name: Path):
+        """Initialize object for the Generic Asset Properties extracted from extra_config."""

```

Comment:
```suggestion
    def __init__(self, file_name: Path):
        """Initialize object for the Generic Asset Properties extracted from extra_config prompt.yaml.
        Args:
            file_name (Path): Prompt config file to load and validate.
        """
```

This should also be added to ModelConfig's init too - I used prompt.yaml above, but please edit if that's the incorrect path

Reply:
Updated, thanks.